apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -18,13 +18,29 @@ export default class AnimationPickerListItem extends React.Component {
     category: PropTypes.string,
     selected: PropTypes.bool,
     isBackgroundsTab: PropTypes.bool,
+    isAnimationJsonMode: PropTypes.bool,
   };
 
   state = {
     loaded: false,
     hover: false,
+    iconImageSrc: null,
   };
 
+  componentDidMount() {
+    let category = this.props.category;
+    if (category) {
+      if (this.props.isAnimationJsonMode && category === 'all') {
+        category = 'all_including_backgrounds';
+      }
```

Comment:
@bencodeorg here's the dynamic import you suggested -thanks!

Reply:
Discussed offline with @bencodeorg - a UI test was consistently failing with this change for unknown reason. And since the code is more complex and no plans to move this component to Storybook, reverting back to using 'blockly/media/...' path.